package.json

```diff
@@ -1,41 +1,55 @@
 {
-  "repository": "graphql/graphql.github.io",
-  "private": true,
-  "version": "0.0.0",
-  "scripts": {
-    "start": "rm -rf build && babel-node resources/server.js",
-    "build": "rm -rf build && babel-node resources/build.js",
-    "watch": "rm -rf build && babel-node resources/watch.js",
-    "test": "npm run build"
-  },
-  "babel": {
-    "optional": [
-      "es7.asyncFunctions",
-      "es7.objectRestSpread"
-    ]
-  },
-  "site": {
-    "source": "./site",
-    "build": "./build"
+  "name": "graphql-docs-site",
+  "description": "GraphQL Docs Site",
+  "version": "0.1.0",
+  "author": "Saihajpreet Singh <saihajpreet.singh@gmail.com>",
```

Comment:
Looking through this again, I don't think we need an `author` field here 🤔

Reply:
My bad 🤦‍♂️ I was using one of my projects to make this and forgot to update. What about version number?

Reply:
Good question, I'm not super familiar with versioning in open source. Wdyt @IvanGoncharov?